client/src/language/utils.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 
-import { Position, Range, type TextDocument } from 'vscode'
+import { Location, type LocationLink, Position, Range, type Uri, type TextDocument } from 'vscode'
```

Comment:
Could you add a debug log with the location of temporary files? Would make it easier to browse them when debugging embedded languages features.

Reply:
done